tests/Makefile.am

```diff
@@ -91,7 +92,8 @@ program_TESTS =					\
 	squash_space				\
 	typeform				\
 	typeform_for_emphclass			\
-	uplow_with_unicode
+	uplow_with_unicode				\
+	table-dev
```

Comment:
Shouldn't this be renamed to `maketable` or similar to make it clear which functionality it is testing?

Reply:
OK makes sense.